pwa/src/templates/categoriesDetailTemplate/CategoriesDetailTemplate.tsx

```diff
@@ -11,15 +11,17 @@ import { TEMPORARY_PORTFOLIOS } from "../../data/portfolio";
 import Skeleton from "react-loading-skeleton";
 import { TEMPORARY_DOMAINS } from "../../data/domains";
 import { TEMPORARY_COMPONENTS } from "../../data/components";
-import { ComponentCard } from "../../components/componentCard/ComponentCard";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faTags } from "@fortawesome/free-solid-svg-icons";
+import { faGripVertical, faLayerGroup, faTable, faTags } from "@fortawesome/free-solid-svg-icons";
+import { FiltersContext } from "../../context/filters";
+import { ComponentResultTemplate } from "../templateParts/resultsTemplates/ComponentResultsTemplate";
 
 interface CategoriesDetailTemplateProps {
   categoryId: string;
 }
 
 export const CategoriesDetailTemplate: React.FC<CategoriesDetailTemplateProps> = ({ categoryId }) => {
+  const [filters, setFilters] = React.useContext(FiltersContext);
   const { t } = useTranslation();
 
   const portfolio = TEMPORARY_PORTFOLIOS.find((category) => {
```

Comment:
A category contains components, not really a portfolio I'd say.

_If_ this template was called `PortfolioDetailTemplate`, I'd expect this variable to be called `portfolioItems` perhaps. However, in this case, I'd probably opt for `components`.

Reply:
a variable called `components` already exists inside this template
and calling it `components` does not make a whole lot of sense considering the data inside `TEMPORARY_PORTFOLIOS`